packages/server/lib/open_project.coffee

```diff
@@ -201,6 +201,9 @@ create = ->
             relaunchBrowser()
       })
 
+      if !_.isUndefined(args.configFile)
+        options.configFile = args.configFile
```

Comment:
this feels off to me... why is this necessary?

Reply:
so that the project gets initialized with the proper `configFile` option, if you remove it, the project won't use the correct `configFile`